chapters/02-estrutura-do-programa.md

```diff
@@ -345,9 +345,11 @@ Usar o operador resto (`%`) é uma maneira fácil de testar se um número é div
 
 A construção do `for` nesse exemplo não contém a parte que checa pelo fim do loop. Isso significa que o loop não vai parar de executar até que a declaração `break` contida nele seja executada.
 
-Se a declaração `break` faltar, ou acidentalmente tivermos uma condição que sempre produza `true`, você terá o chamado *loop infinito*. Um programa rodando um loop infinito nunca vai parar de rodar, que é normalmente uma coisa ruim.
+Se você não incluir a declaração `break` ou acidentalmente escrever uma condição que sempre produza um resultado `true`, seu programa ficará preso em um _loop infinito_. Um programa preso em um loop infinito nunca vai terminar sua execução, o que normalmente é uma coisa ruim.
```

Comment:
Acho interessante deixar o itálico sim, pois assim o leitor fica mais atento ao termo.